### ISSUE TITLE
Phone Dialer and Number formating

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#78]
```
There isn't any phone dialpad to enter manual dial numbers. You also need to 
remove the () from the area code and changes sending number format. (i.e. 
sip:(321)230-0000@voipserver.com  needs to be sent as 
sip:3212300000@voipserver.com. When in this format calls fail because proxies 
don't remove these characters. IMSDroid/v1.0.302
```

Original issue reported on code.google.com by `voiptele...@gmail.com` on 16 Oct 2010 at 6:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 18 Oct 2010 at 4:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 1.0.311.
```

Original comment by `boss...@yahoo.fr` on 19 Oct 2010 at 10:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 26 Oct 2010 at 9:33
- Changed state: **Fixed**
